src/mscorlib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -8,6 +8,12 @@
 using System.Diagnostics.Contracts;
 using System.Collections.Generic;
 
+#if INSIDE_CLR
+using Kernel32 = Interop.Kernel32;
```

Comment:
You do not need these. The CoreRT copy should be switched to kernel32.dll, etc. as well. https://github.com/dotnet/corert/issues/2241